react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+export default function ToggleSwitch() {
+  const [toggle, setToggle] = useState(false);
```

Comment:
`toggle` isn't a very good name because it doesn't indicate what the state value represents. Looking at your code, a value of `true` means it is switched on. So maybe `isOn` would be a better name?